src/CADL.Extension/Emitter.Csharp/src/type/CodeModelType.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { InputType } from "./InputType";
+import { InputTypeKind } from "./InputTypeKind";
+import { InputTypeSerializationFormat } from "./InputTypeSerializationFormat";
+import { InputTypeValue } from "./InputTypeValue";
+
+// ToDO: need to refine
+export interface Schema {
+    Name: string;
+}
+export class CodeModelType implements InputType {
```

Comment:
We would never generate `CodeModelXxx` types, they are for compatibility with CodeModel on autorest.csharp side

Reply:
I will remove.